src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -525,6 +525,29 @@ def accessdb_config_key(cls, version=None):
             version = cls.VERSION
         return "{}{}".format(cls.ACDB_CONFIG_KEY, version)
 
+    def check_and_update_db(self):
+        logger.debug("Checking for previews DB versions")
```

Comment:
I think it's:

```suggestion
        logger.debug("Checking for previous DB versions")
```

Reply:
Good catch, copy & paste error :-)

Reply:
Yes... right. It was introduced by the removed code.